repose-aggregator/commons/utilities/src/main/java/org/openrepose/commons/utils/servlet/http/MutableHttpServletResponse.java

```diff
@@ -302,11 +302,16 @@ public void sendError(int code) throws IOException {
 
     @Override
     public void sendError(int code, String message) {
-        super.setStatus(code);
-        this.message = message;
+        setStatus(code, message);
         error = true;
     }
 
+    @Override
+    public void setStatus(int code, String message) {
+        super.setStatus(code, message);
+        this.message = message;
```

Comment:
Since the original single parameter `sendError()` method was calling the single parameter `super.setStatus()` method instead of the single parameter `super.sendError()` method I opted to do the same for the two parameter flavor in an effort to minimize the behavior change as much as possible until we can get the new wrappers in place.

Yes, the two parameter flavor of the `setStatus()` method is deprecated *"due to ambiguous meaning of the message parameter"*, but it is this ambiguity that is biting us to begin with.

If we didn't do the write to the body, then the `ResponseMessageService` configuration could also use the *"IF_EMPTY"* override again. We would just have to provide a default catch-all `status-code` element in the `response-messaging.cfg.xml` that contained:
```
    <status-code id="Default" code-regex="\d\d\d" overwrite="IF_EMPTY">
        <message media-type="*/*">
            %M
        </message>
    </status-code>
```
But I will leave that for another story.